contracts/token/OtcEscrow.sol

```diff
@@ -0,0 +1,91 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.6.10;
+
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
+import { SafeMath } from "@openzeppelin/contracts/math/SafeMath.sol";
+
+import { Vesting } from "./Vesting.sol";
+
+/*
+    Simple OTC Escrow contract to transfer index in exchange for specified USDC amount
```

Comment:
There should be a * at the beginning of this line. Make sure they all lineup as well. You can find examples in the `ICManager` for formatting these javadocs